packages/core/admin/server/middlewares/rateLimit.js

```diff
@@ -8,18 +8,30 @@ const { RateLimitError } = utils.errors;
 module.exports =
   (config, { strapi }) =>
   async (ctx, next) => {
-    const ratelimit = require('koa2-ratelimit').RateLimit;
-
-    const userEmail = toLower(ctx.request.body.email) || 'unknownEmail';
-
-    return ratelimit.middleware({
-      interval: { min: 5 },
-      max: 5,
-      prefixKey: `${userEmail}:${ctx.request.path}:${ctx.request.ip}`,
-      handler() {
-        throw new RateLimitError();
-      },
-      ...strapi.config.get('admin.ratelimit'),
-      ...config,
-    })(ctx, next);
+    let ratelimitConfig = strapi.config.get('admin.ratelimit');
+
+    if (!ratelimitConfig || !ratelimitConfig.enabled) {
```

Comment:
I think you wanted something like that no?
```suggestion
    if (!ratelimitConfig || !has('enabled', ratelimitConfig)) {
```

or 

```suggestion
    if (!ratelimitConfig || !('enabled' in ratelimitConfig)) {
```

because if `enabled = false`, it would enter the if and put `enabled = true` :p